tests/smoke-test/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "./deployment",
```

Comment:
there is no `./deployment` directory, I suppose this is just a placeholder since there are no ts files to be transpiled?

Reply:
it is being created on compile. not sure if this is the best practice, but the code is copied from the sample code, therefore it should not be duplicated.

Reply:
I see, I think I would prefer it if we use the existing files instead and adapt the package.json scripts accordingly